carAPI/src/main/java/com/everis/boundary/CarResource.java

```diff
@@ -21,26 +22,23 @@
 import javax.ws.rs.core.Response.Status;
 import javax.ws.rs.core.UriInfo;
 
-import org.apache.log4j.Logger;
-
 import com.everis.control.CarService;
 import com.everis.entity.Car;
+import com.everis.utils.LoggerInterceptor;
 
 
 @Path("/cars")
 @Produces(MediaType.APPLICATION_JSON)
 @Consumes(MediaType.APPLICATION_JSON)
+@Interceptors(LoggerInterceptor.class)
 public class CarResource implements CarResourceInterface{
 	
-	private static final Logger LOG = Logger.getLogger(CarResource.class);
```

Comment:
It is not really necessary to remove the custom logs from your classes. With the interceptor, we add generic logs for all methods, but sometimes is good to have more specific ones 😄 ( it is up to you )